src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -126,6 +127,17 @@ class ExplorerTable extends React.Component {
             return rowComp;
           }
         }
+
+        // if this field is the `dicomViewerId`, convert the value to a link to the DICOM viewer
+        if (this.props.tableConfig.dicomViewerId && this.props.tableConfig.dicomViewerId === field && valueStr) {
```

Comment:
make sense to put this section into the `// handling some special field types` section down below?

Reply:
i tried it and it works too but doesn't look right IMO. I chose to separate it so the code would be less confusing and easier to maintain. if we move this to the `switch` statement:
- the order of the terms in the "if" cannot change (because of how "switch" compares the value of `field`, for example `this.props.tableConfig.dicomViewerId &&` can't be first)
- the order of the switch cases cannot change (because this new logic relies on the "linkFields" case to run afterwards to transform the value into a link)

Reply:
i can change it if you think it's cleaner

Reply:
oh good point, don't worry about this then